src/Router.tsx

```diff
@@ -123,9 +123,12 @@ const router = createBrowserRouter([
         path: paths.HOME,
         element: (
           <Auth>
-            <Outlet />
-            <Home />
-            <CommandBar />
+            <ModelingMachineProvider>
+              <Outlet />
+              <Home />
+              <CommandBar />
+            </ModelingMachineProvider>
+            <WasmErrBanner />
```

Comment:
Noticed that this work only when you open the app signed in. Works after a refresh after signing in, which is probably why the test fails

Reply:
Yeah [forcing a refresh after auth](https://github.com/KittyCAD/modeling-app/pull/2661/files#diff-b104af957bda276a1f3e77c8c4bf09dcce1e262c0667c0ef10008f8a27e546c3R73) makes the test happy.

@lf94 @franknoirot The way I tried to enable the engine connection on the home page here was a complete shot in the dark. Let me know how you'd want to proceed

Reply:
Also, sorry for the dirty diff here. We should merge https://github.com/KittyCAD/modeling-app/pull/2659 first